SyncUps/SyncUps/SyncUpsApp.swift

```diff
@@ -3,6 +3,8 @@ import SwiftUI
 
 @main
 struct SyncUpsApp: App {
+  static let model = AppModel(syncUpsList: SyncUpsListModel())
```

Comment:
This is the pattern we've used elsewhere. I thought it might also prevent over-observation when properties are accessed in the model's `init`, but on second thought it's lazy, so maybe not.